internal/postprocessor/main.go

```diff
@@ -15,64 +15,123 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"flag"
+	"fmt"
 	"io"
 	"io/fs"
+	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"cloud.google.com/go/internal/gapicgen/generator"
+	"cloud.google.com/go/internal/gapicgen/git"
+	"cloud.google.com/go/internal/gensnippets"
+	"cloud.google.com/go/internal/postprocessor/execv"
 	"cloud.google.com/go/internal/postprocessor/execv/gocmd"
+	"golang.org/x/sync/errgroup"
+	"gopkg.in/yaml.v2"
 )
 
 func main() {
-	srcRoot := flag.String("src", "owl-bot-staging/src/", "Path to owl-bot-staging directory")
-	dstRoot := flag.String("dst", "", "Path to clients")
+	var stagingDir string
+	var clientRoot string
+	var googleapisDir string
+	var directories string
+	flag.StringVar(&stagingDir, "stage-dir", "owl-bot-staging/src/", "Path to owl-bot-staging directory")
+	flag.StringVar(&clientRoot, "client-root", "/repo", "Path to clients")
+	flag.StringVar(&googleapisDir, "googleapis-dir", "", "Path to googleapis/googleapis repo")
+	flag.StringVar(&directories, "dirs", "", "Comma-separated list of modules to run")
```

Comment:
Maybe I am missreading something, what happens if this flag is not set, will the code still run? If not, I think it should -- for everthing.

Reply:
Yes, it will run on all modules if not passed, as default value is empty string and that will set config.modules field to 'nil'

Reply:
Please see lines 58 and 95